.github/workflows/conformance-datapath.yaml

```diff
@@ -266,7 +272,8 @@ jobs:
             ./contrib/scripts/kind.sh "" 3 "" "" "${{ matrix.kube-proxy }}"
             ./cilium-cli install ${{ steps.vars.outputs.cilium_install_defaults }} \
               --helm-set-string="kubeProxyReplacement=${{ matrix.kpr }}" \
-              --encryption=${{ matrix.encryption }}
+              --encryption=${{ matrix.encryption }} \
+               --helm-set-string="encryption.nodeEncryption=${{ matrix.encryption-node }}"
```

Comment:
Two commits before, there's this docs change that says we can use a nice `--node-encryption` flag. Is that not possible here because we are using an outdated CLI version?

Reply:
It should be available once we update to cilium-cli v0.12.12: https://github.com/cilium/cilium/pull/23030

Reply:
I think we should be able to use that by now. Updated.